torch/_dynamo/config.py

```diff
@@ -352,6 +352,8 @@ def is_fbcode():
 _experimental_support_context_fn_in_torch_utils_checkpoint = False
 
 
+output_bytecode_hooks = []
```

Comment:
Add a docstring above this explaining what it does

Reply:
Sure, will do.